unity-ktwo/Assets/Scripts/Behaviours/DishWashBehaviour.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DishWashBehaviour : MonoBehaviour
+{
+    [Header("Dish Wash Settings")]
+    public float speedDebuffPercent;
+    public float duration;
+    public float abilityDuration;
+
+    private GameObject player;
+    private Vector3 offsetFromPlayer;
+    private bool active = true;
+    private HashSet<GameObject> affectedEntities = new HashSet<GameObject>();
+
+    void Awake()
+    {
+        player = PlayerManager.instance.player;
```

Comment:
race condition: accessing the static instance from a manager should happen earliest at `Start` because instance gets initialized on `Awake`

Reply:
fixed